Upstream: submitted
Source: https://marc.info/?l=netfilter-devel&m=146907333005540&w=2

From 41bbda5002e638a51eaca4cadcbbf9de659c251e Mon Sep 17 00:00:00 2001
From: Kylie McClain <somasis@exherbo.org>
Date: Mon, 18 Jul 2016 12:48:55 -0400
Subject: [PATCH] Fix test building on musl libc

On musl libc, sys/select.h is where FD_* definitions are located.

Signed-off-by: Kylie McClain <somasis@exherbo.org>
---
 examples/netfilter/nfct-daemon.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/examples/netfilter/nfct-daemon.c b/examples/netfilter/nfct-daemon.c
index a97c2ec..d223ac2 100644
--- a/examples/netfilter/nfct-daemon.c
+++ b/examples/netfilter/nfct-daemon.c
@@ -13,6 +13,7 @@
 #include <netinet/in.h>
 #include <inttypes.h>
 #include <errno.h>
+#include <sys/select.h>
 
 #include <libmnl/libmnl.h>
 #include <linux/netlink.h>
-- 
2.9.0

